src/kixi/nippy/tools.clj

```diff
@@ -0,0 +1,48 @@
+(ns taoensso.nippy.tools
```

Comment:
Does this ns want to change? If not, why not?

Reply:
Not sure this is actually used by anything, came over as part of the nippy copy